pkg/image/fetcher.go

```diff
@@ -109,7 +110,17 @@ func (f *Fetcher) Fetch(ctx context.Context, name string, options FetchOptions)
 	}
 
 	f.logger.Debugf("Pulling image %s", style.Symbol(name))
-	err = f.pullImage(ctx, name, options.Platform)
+	platform := options.Platform
+	for i := 0; i <= 1; i++ {
```

Comment:
The for loop is not really needed, ist it? I think it overcomplicates things. How about something like:

```
err = f.pullImage(ctx, name, platform)

if(errors.Is(err, ErrPlatformNotMatch)) {
  err = f.pullImage(ctx, name, "")
}
```

Reply:
Thank you @colincasey